web/templates/compare.html

```diff
@@ -4,7 +4,7 @@
 {% block content %}
         <div class="container">
             <div class="row mt-5">
-                <h2 class="col-12">Comparing {{ lang1_friendlyname }}'s and {{ lang2_friendlyname }}'s {{ concept }}</h2>
+                <h2 class="col-12">Comparing {{ lang1_friendlyname|linebreaksbr }}'s and {{ lang2_friendlyname|linebreaksbr }}'s {{ concept|linebreaksbr }}</h2>
```

Comment:
Same here. The name of the variable shouldn't ever be more than one line. (it's just things like "Python" or "JavaScript")